### PR TITLE
add task controller

### PR DESCRIPTION
**Pull request by Supermalonz**

### CODE REVIEW

File: app/views/tasks/show.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello
```
Comment (dolwijit13) [inline app/views/tasks/show.html.erb:1]: 🙈

Comment (Supermalonz) [reply app/views/tasks/show.html.erb:1]: my bad 🙈

File: db/migrate/20220513024820_create_tasks.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreateTasks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :tasks do |t|
+      t.string :title
+      t.text :body
```
Comment (t-kanin) [inline db/migrate/20220513024820_create_tasks.rb:5]: add `null: false` to both title and body

File: app/controllers/tasks_controller.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    end
+  end
+
+  def edit; end
+
+  def update
+    if @task.update(task_params)
+      redirect_to tasks_path
+    else
+      render :edit
```
Comment (t-kanin) [inline app/controllers/tasks_controller.rb:32]: here as well

File: app/controllers/tasks_controller.rb [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  def new
+    @task = Task.new
+  end
+
+  def create
+    @task = current_user.tasks.new(task_params)
+    if @task.save
+      redirect_to tasks_path
+    else
+      render :new
```
Comment (t-kanin) [inline app/controllers/tasks_controller.rb:22]: show flash message when can't update ex. flash[:error] = @task.errors.full_messages

File: app/models/task.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+class Task < ApplicationRecord
+  validates :title, presence: true
+  validates :body, presence: true
+  validates :status, inclusion: [true, false]
```
Comment (t-kanin) [inline app/models/task.rb:4]: actually `status` could be better with enum. 

https://blog.saeloun.com/2022/01/05/how-to-use-enums-in-rails.html

**Comment by t-kanin** [comment]
Don't forget to add PR description na kub

**Pull request merged**